### ISSUE TITLE
Add term talipes varus

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10520]
**Preferred term label:**
Talipes varus

**Synonyms**
Pes varus, Pes adductus

**Definition (free text, please give PubMed ID)**
Inversion of the foot, resulting in the outer side of the sole only touching the ground; usually some degree of talipes equinus is associated with it, and often talipes cavus. https://medical-dictionary.thefreedictionary.com/talipes+varus

**Parent term (use hpo.jax.org/app)**
Abnormal ankle morphology HP:0034673

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
An example in chondrodysplasia with joint dislocations, gPAPP type (omim614078) ,PMID22887726
Part of clubfoot (talipes equinovarus)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl
cirrently we have this listed as a synonym of [Talipes equinovarus HP:0001762](https://hpo.jax.org/browse/term/HP:0001762)
Is it correct that Pes varus is a component of Talipes equinovarus and we should remove the synonym?

**Comment by azankl** [comment]
@pnrobinson  Yes, they are not synonyms, see picture below:
![talipes](https://github.com/user-attachments/assets/0cc5224d-46c8-4c1e-88e4-56d4ca1bac37)


[Medical Dictionary](https://medical-dictionary.thefreedictionary.com/talipes) also has a better definition:
a deformity in which the foot is twisted out of normal position; see also [clubfoot](https://medical-dictionary.thefreedictionary.com/clubfoot) and see illustration. It may have an abnormally high longitudinal arch (talipes cavus) or it may be in dorsiflexion (talipes calcaneus), in plantar flexion (talipes equinus), abducted and everted (talipes valgus or [flatfoot](https://medical-dictionary.thefreedictionary.com/flatfoot)), adducted and inverted (talipes varus), or various combinations of these (talipes calcaneovalgus, talipes calcaneovarus, talipes equinovalgus, or talipes equinovarus). 

Please note that Medical Dictionary has a similar picture, but it is actually wrong (they have mislabelled talipes equinovarus as talipes equinovalgus)

So ideally, we should add the 9 subtypes shown in the picture under `Talipes HP:0001883` and change the definition to something like the above.

**Comment by pnrobinson** [comment]
added Talipes varus as 
HP:0025802

I thiink the subhierarchy could use some care, maybe we can meet up at some future ASHG to go over.
